luaui/Widgets/cmd_exclude_walls_area_set_target.lua

```diff
@@ -0,0 +1,76 @@
+local widget = widget ---@type Widget
+
+function widget:GetInfo()
+	return {
+		name 		= "Exclude walls from area set target",
+		desc 		= "Stops walls from being included in area set target if units are present",
+		date		= "April 2025",
+		license 	= "GNU GPL, v2 or later",
+		layer 		= 0,
+		enabled 	= true
+	}
+end
+
+local excludedUnitsDefID = {}
+local CMD_SET_TARGET = 34923
+local spGetUnitDefID = Spring.GetUnitDefID
+
+local gameStarted
+
+for id, unitDef in pairs(UnitDefs) do
+	if unitDef.customParams.objectify then
+		table.insert(excludedUnitsDefID, id)
```

Comment:
Make it a map instead of array
```suggestion
		excludedUnitsDefID[id] = true
```